tests/common.rs

```diff
@@ -1,9 +1,35 @@
+use std::env;
+
 pub const ROOT_USERNAME: &str = "root";
 pub const ROOT_PASSWORD: &str = "KWNngteTps7XjrNv";
 
 pub const NORMAL_USERNAME: &str = "username";
 pub const NORMAL_PASSWORD: &str = "password";
 
+pub fn get_root_user() -> String {
+    let root_user = env::var("ARANGO_ROOT_USER").expect("Root user not set");
```

Comment:
What about using the `ROOT_USERNAME` const as default?

Reply:
Oh, I see. You would like to use the const in case no environment variable was defined, right?

Reply:
Yes. But I wondering if this is good idea. With default value, these getter functions is infallable, which might sounds good. 

But if we forget to set these environment variables, we may encounter a inconsistent values in arangodb and arangors. I use docker for tests and CI, we could pass username and password once in environment variables and let docker container, travis and arangors inherit used env variables.